edx_api/email_settings_test.py

```diff
@@ -0,0 +1,38 @@
+"""test for change email settings"""
```

Comment:
Just a general question, I believe the test case for `Optout` object actually getting deleted in case of unsubscribing would have been part of Open Edx. Am I correct?

Reply:
yeah, Optout creation and deletion are edX parts.